clusterloader/run-e2e.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
You need this copyright boilerplate, as e.g. in:
https://github.com/kubernetes/perf-tests/blob/master/verify/test.sh